### PR TITLE
fix: update beets treasuries

### PR DESCRIPTION
**Pull request by gosuto-inzasheru**
based on franz's comments here: https://forum.balancer.fi/t/bip-xxx-linear-pool-hack-restituion/5541/3?u=gosuto

### CODE REVIEW

**Comment by gosuto-inzasheru** [comment]
extra ref: https://docs.beets.fi/ecosystem/treasury

**Comment by gosuto-inzasheru** [comment]
do we know what `0x811912c19eef91b9dc3ca52fc426590cfb84fc86` should be labelled as?

File: extras/multisigs.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "blabs_ops": "0x02f35dA6A02017154367Bc4d47bb6c7D06C7533B",
     "linearPoolController": "0x75a52c0e32397A3FC0c052E2CeB3479802713Cf4",
     "foundation_opco": "0x3B8910F378034FD6E103Df958863e5c684072693",
-    "beets_treasury": "0x811912c19eEF91b9Dc3cA52fc426590cFB84FC86",
+    "beets_treasury": "0xea06e1b4259730724885a39ce3ca670efb020e26",
```
Comment (Tritium-VLK) [inline extras/multisigs.json:13]: This old beets treasury address is also a beets multisig. It's where we send them money every motnh to pay people.

I guess that is different from the DAO treasury.

It's also a little unclear where we shold put multisigs we don't have any signers on.  Maybe they make more sense in extras/beets or extras/partnerMsigs or someing.

At very least I suggest renaming the old beets_treasury to beets_sp_inbound or something.

Also any context where you got the new addresses from, esp the second one tha tlooks to be on op or something.

**Review by Tritium-VLK — CHANGES_REQUESTED** [review]

File: extras/multisigs.json (hunk 1 above)
Comment (gosuto-inzasheru) [reply extras/multisigs.json:14]: i added the old one back in as `beets_service_provider`

> Also any context where you got the new addresses from, esp the second one tha tlooks to be on op or something.

please read my comments

**Review by Tritium-VLK — APPROVED** [review]

**Pull request merged**